pgraph/pgraph.go

```diff
@@ -231,6 +221,25 @@ func (g *Graph) VerticesChan() chan Vertex {
 	return ch
 }
 
+// Vertices returns a randomly sorted slice of all vertices in the graph.
+// The order is random, because the map implementation is intentionally so!
+func (g *Graph) Vertices() []Vertex {
+	var vertices []Vertex
+	for k := range g.adjacency {
+		vertices = append(vertices, k)
+	}
+	return vertices
+}
+
+// EdgeSlice is a linear list of edges. It can be sorted.
+type EdgeSlice []Edge
+
+func (es EdgeSlice) Len() int { return len(es) }
```

Comment:
These need the standard comments for golint to pass. You can use the ones from the existing VertexSlice.